packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/defaultmodules/ErrorManagerModule.kt

```diff
@@ -0,0 +1,25 @@
+package expo.modules.kotlin.defaultmodules
+
+import android.os.Bundle
+import expo.modules.kotlin.exception.CodedException
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+
+private const val onNewException = "SweetErrorManager.onNewException"
+
+class ErrorManagerModule : Module() {
+  override fun definition() = ModuleDefinition {
+    name("SweetErrorManager")
```

Comment:
I guess we should be more explicit in naming 🤔 
First thing: `Expo` prefix - are we dropping it? If not, let's be consistent with it.
Second thing: `SweetErrorManager` suggests it's just some error manager that is **sweet** (meaning it's simply nice to use), but it should rather suggest it's specifically tailored for **Sweet API**, am I right?
```suggestion
    name("ExpoSweetAPIErrorManager")
```

Reply:
With Łukasz we've discussed about related thing — my concern is that we shouldn't use the SweetAPI as a name in our code. We can use it to differentiate the old architecture from the new one, but our goal is to entirely migrate to the new one, so that name would become unnecessary 🤔

Reply:
As this process will take a while I guess there's no problem using it for a time being 🤔 
Moreover being a little more explicit in naming is not a bat thing 🤷 
We can always refactor at the later point in time as this API is constantly evolving and mutating (we're still at `0.x.x` version) 😉